tests/test_indicators.py

```diff
@@ -488,6 +488,9 @@ def test_all_parameters_understood(official_indicators):
     if problems - {
         ("COOL_NIGHT_INDEX", "lat"),
         ("DRYNESS_INDEX", "lat"),
+        # TODO: How should we handle the case of Literal[str]?
+        ("GROWING_SEASON_END", "op"),
+        ("GROWING_SEASON_START", "op"),
```

Comment:
I like the idea of `Literal`. 

I think we could simply map it to xclim's STRING ? Unless there's a simple way to detect if the allowed values' type, my guess is that we don't really need Literals for anything else than strings. So here : https://github.com/Ouranosinc/xclim/blob/98c251cf59f5e9c0e1573593342445e08ea9c957/xclim/core/utils.py#L659-L660

And if we do that, maybe we could apply the same for all `op` and `method`  arguments throughout the library ? And maybe use the annotation to parse the `choices` property of `xclim.core.indicator.Parameter` ?